### ISSUE TITLE
Stored Variables Regression

### ISSUE THREAD

**Issue by calebd** [krzysztofzablocki/Sourcery#334]
It looks like Type.storedVariables now includes both `private` and `fileprivate` members. This breaks the included `AutoEquatable` template. I tried adding accessibility filters which always resulted in an empty variables array.

**Comment by ilyapuchka** [comment]
To filter variables by their access level there are `privateGet` and `privateSet` and others filters ([docs](https://cdn.rawgit.com/krzysztofzablocki/Sourcery/master/docs/writing-templates.html)):

```
{% for variable in variables|!privateGet|!fileprivateGet %}
```

or you can use `where`: 

```
{% for variable in variables where variable.readAccess != "private" and variable.readAccess != "fileprivate" %}
```

@Antondomashnev should we check other templates for such regression? Do you mind doing that? You could also maybe look at #342

**Comment by Antondomashnev** [comment]
@ilyapuchka sure 👌

**Comment by Antondomashnev** [comment]
@calebd could you please clarify the prior behavior? Did it include only `public` and `internal` variables?

**Comment by ilyapuchka** [comment]
@Antondomashnev yes, we changed recently that private declarations are not filtered out any more (https://github.com/krzysztofzablocki/Sourcery/pull/201) with introduction of access level filters

**Comment by calebd** [comment]
In `0.5.9`, `storedVariables` included both `public` and `internal` and included variables that had both `get` and `set` blocks. I relied on this behavior for my own `SourceryEquatable` template.

I have a type like this:

```
struct Thing {
    fileprivate _nested: Box<Thing>?

    var nested: Thing? {
        get { return _nested.value }
        set { _nested = newValue.map(Box.init) }
    }
}
```

Old Sourcery output:

```
func == (lhs: Thing, rhs: Thing) -> Bool {
    if lhs.nested != rhs.nested { return false }
    return true
}
```

New Sourcery output with no template changes:

```
func == (lhs: Thing, rhs: Thing) -> Bool {
    if lhs._nested != rhs._nested { return false }
    return true
}
```

 I have yet to find a set of filters or variables that mirrors that behavior

**Comment by Antondomashnev** [comment]
@ilyapuchka @calebd thanks, I've opened the PR that should fix an issue with updated templates.

**Comment by ilyapuchka** [comment]
Solved with #345
